dokka-subprojects/plugin-base/src/main/kotlin/org/jetbrains/dokka/base/signatures/KotlinSignatureProvider.kt

```diff
@@ -255,6 +257,7 @@ public class KotlinSignatureProvider(
             ?: false
     }
 
+    @OptIn(ExperimentalDokkaApi::class)
```

Comment:
minor: as we do opt-in on a class itself, it’s unnecessary to repeat opt-in in nested scopes